integration-tests/src/test/scala/tech/beshu/ror/integration/suites/SearchApiSuite.scala

```diff
@@ -204,13 +271,69 @@ object SearchApiSuite {
   private def nodeDataInitializer(): ElasticsearchNodeDataInitializer = (esVersion, adminRestClient: RestClient) => {
     val documentManager = new DocumentManager(adminRestClient, esVersion)
     val indexManager = new IndexManager(adminRestClient, esVersion)
+    val templateManager = new IndexTemplateManager(adminRestClient, esVersion)
+    val dataStreamManager = new DataStreamManager(adminRestClient, esVersion)
+
+    createDataStream(templateManager, dataStreamManager, "test_logs_ds", "test_logs_ds_index_template")
+    createDocsInDataStream(documentManager, indexManager, "test_logs_ds", 5)
 
-    createDataStreamWithAlias(indexManager, documentManager)
+    indexManager
+      .updateAliases(
+        AliasAction.Add(index = "test_logs_ds", alias = "alias_ds"),
+      )
+      .force()
+
+    createOldFashionedDataStream(indexManager, documentManager)
     createRealLifeTestsDocumentsAndAliases(indexManager, documentManager)
   }
 
-  private def createDataStreamWithAlias(indexManager: IndexManager,
-                                        documentManager: DocumentManager) = {
+  private def createDataStream(adminTemplateManager: IndexTemplateManager,
+                               adminDataStreamManager: DataStreamManager,
+                               dataStreamName: String,
+                               indexTemplateName: String): Unit = {
+    adminTemplateManager.createTemplate(indexTemplateName, indexTemplate(dataStreamName)).force()
+    adminDataStreamManager.createDataStream(dataStreamName)
+  }
+
+  private def indexTemplate(dataStreamName: String) = ujson.read(
```

Comment:
It seems we have the same (or similar) in the DataStreamApiSuite. I understand that creating a data stream is not one operation (from the ES API perspective). But for the sake of our tests, it'd be nice to have an util which encapsulates all the methods under one: `def createDataStream(name: String)`. WDYT about creating such manager?